stencil.config.ts

```diff
@@ -40,6 +40,10 @@ export const config: Config = {
     allowableMismatchedPixels: 0,
     moduleDirectories: ['node_modules', './'],
     timers: 'fake',
+    // browserHeadless: false,
+    // browserDevtools: true,
+    // browserSlowMo: 1000, // milliseconds
+    browserArgs: ['--no-sandbox', '--disable-setuid-sandbox'],
```

Comment:
Looks like you accidentally committed these lines?

Reply:
Actually only the browser args. I'll leave the other ones in there because they are helpful when debugging e2e tests.